widget.py

```diff
@@ -45,6 +45,29 @@ def __init__(self, parent=None):
         self.tank_mass_points = np.empty((0,2))
         self.engine_thrust_points = np.empty((0,2))
 
+        # Also need to connect to data to change accordingly
+        # self.ui.quickDisconnectState.setStyleSheet("background-color: rgb(255, 80, 80)") #edited
+        # self.ui.cv1State.setStyleSheet("background-color: rgb(255, 80, 80)")
+        # self.ui.xv1State.setStyleSheet("background-color: rgb(255, 80, 80)")
+        # self.ui.xv2State.setStyleSheet("background-color: rgb(255, 80, 80)")
+        # self.ui.xv3State.setStyleSheet("background-color: rgb(255, 80, 80)")
+        # self.ui.xv4State.setStyleSheet("background-color: rgb(255, 80, 80)")
+
+
+        """ connect them with the data
+        self.ui.quickDisconectState.setStyleSheet("background-color: rgb(80, 255, 80)") #edited
+        self.ui.quickDisconectState.setText("ON")
+        self.ui.cv1State.setStyleSheet("background-color: rgb(80, 255, 80)")
+        self.ui.cv1State.setText("ON")
+        self.ui.xv1State.setStyleSheet("background-color: rgb(80, 255, 80)")
+        self.ui.xv1State.setText("ON")
+        self.ui.xv2State.setStyleSheet("background-color: rgb(80, 255, 80)")
+        self.ui.xv2State.setText("ON")
+        self.ui.xv3State.setStyleSheet("background-color: rgb(80, 255, 80)")
+        self.ui.xv3State.setText("ON")
+        self.ui.xv4State.setStyleSheet("background-color: rgb(80, 255, 80)")
+        self.ui.xv4State.setText("ON")
+        """
```

Comment:
Did you mean to set the background color to red and to leave this here? It might be nice to have the background color be red initially